lib/netconf/netconf.go

```diff
@@ -175,8 +175,36 @@ func Load(path string) (Network, error) {
 	return net, nil
 }
 
+// Below is used for a temprorary hack to shuffle the holesky rpcs
+
+var (
+	holeksyRPCs = []string{
+		"https://ethereum-holesky-rpc.publicnode.com",
+		"https://rpc.holesky.ethpandaops.io",
+		"https://1rpc.io/holesky",
+		"https://holesky.drpc.org",
+	}
+)
+
+// nextHoleskyRPC cycles through the public Holesky RPCs.
+func nextHoleskyRPC() string {
+	rpcs := holeksyRPCs
+	holeksyRPCs = append(holeksyRPCs[1:], holeksyRPCs[0])
```

Comment:
I haven't tested this but it doesn't look thread safe. Would this still work when called concurrently?